WiFlyHQ.h

```diff
@@ -79,47 +79,47 @@ typedef const char PROGMEM prog_char;
 #endif
 
 /* IP Protocol bits */
-#define WIFLY_PROTOCOL_UDP		0x01
-#define WIFLY_PROTOCOL_TCP		0x02
-#define WIFLY_PROTOCOL_SECURE		0x04
-#define WIFLY_PROTOCOL_TCP_CLIENT	0x08
-#define WIFLY_PROTOCOL_HTTP		0x10	/* HTTP Client mode */
-#define WIFLY_PROTOCOL_RAW		0x20
-#define WIFLY_PROTOCOL_SMTP		0x40
+#define WIFLY_PROTOCOL_UDP        0x01
```

Comment:
they aligned within their groups.  they were most certainly not aligned in any consistent way before.  some had spaces, some had tabs.